src/archive/2023/2023-09-21.md

```diff
@@ -0,0 +1,51 @@
+# Sep - Keegan Korthauer
+
+!!! info "Event Details"
+
+    **Date/Time:**
+
+    Thursday, September 21st, 2023 :material-clock: 11:30am - 1:30pm PT
+
+    **Location:**
+
+    :material-map-marker: SFU Big Data Hub, room ASB 10900
+
+    :material-video: UBC, Michael Smith Laboratories, MSL 102 (Live Stream Location)
+
+<!-- **RSVP:**
+
+If you are interested in attending this seminar *in person*, please fill out [the RSVP form].
+
+**Zoom Meeting:**
```

Comment:
Remove ZOOM meeting info. We aren't sharing publicly this season, just by special request.

Reply:
done